src/pushsource/_impl/model/vms.py

```diff
@@ -58,9 +58,10 @@ class VMIPushItem(PushItem):
     """A brief human-readable description of the image."""
 
     @release.validator
-    def __validate_release(self, attribute, value):
-        # pylint: disable=unused-argument
-        if value and not type(value) == self._RELEASE_TYPE:
+    def __validate_release(self, attribute, value):  # pylint: disable=unused-argument
+        if (
+            value and type(value) is not self._RELEASE_TYPE
```

Comment:
I believe it could actually be made idiomatic, I don't know a specific reason why we need to enforce a precise release type rather than `isinstance`, and there are some downsides to the precise check - e.g. I don't know a reason why a test should be blocked from providing its own subclass with extra testing features.

I noticed it as a minor issue during code review of this recently-introduced code.  I may have forgotten to mention it or didn't bother since that review was already taking a long time.

Reply:
I did try to use `isinstance()` at first, but that made the [test_vmi_release()](https://github.com/release-engineering/pushsource/blob/09dc42211195d153c85624845e15b7c4b4023b9f/tests/model/test_vhd.py#L39-L68) fail as the `AmiRelease`, which is a subclass of the `VMIRelease` is not expected to be associated with `VHDPushItem`. The docstring "_Ensure that subclasses of VMIRelease can not be associated with VHDPushItem_" suggests that unidiomatic strict type check is the expected one as well.